js/animation/fx.d.ts

```diff
@@ -15,10 +19,10 @@ export interface animationConfig {
     /**
      * @docid
      * @type_function_param1 $element:DxElement
-     * @type_function_param2 config:object
+     * @type_function_param2 config:animationConfig
```

Comment:
Better leave `object` here

```suggestion
     * @type_function_param2 config:object
```